pkg/compat/conversion/batchv1beta/conversion_generated.go

```diff
@@ -0,0 +1,261 @@
+// +build !ignore_autogenerated
```

Comment:
A k8s conventions seems to be to prefix generated files with `zz_` should we do that here?

Reply:
That could be adjusted via argument. Will fix it now

Reply:
@jortel Added `zz_` as a prefix